.github/workflows/integration-tests.yml

```diff
@@ -35,14 +31,9 @@ jobs:
 
   dispatch-trivy:
     needs: [ build ]
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Fire off Trivy action
-        uses: broadinstitute/workflow-dispatch@v1
-        with:
-          workflow: Trivy
-          token: ${{ secrets.BROADBOT_TOKEN }}
+    uses: ./.github/workflows/trivy.yml
```

Comment:
I think the point of running trivy as its own workflow was so that its failure didn't fail the build, because all trivy failures we've seen so far were issues with the base image and out of our control.

If it's run this way, will it fail the build?

Reply:
I think it is run as a separate stage of the workflow, so it will still run the other set of steps.